src/kaleidoscope/plugin/IdleLEDs.cpp

```diff
@@ -21,12 +21,20 @@
 namespace kaleidoscope {
 namespace plugin {
 
-uint16_t IdleLEDs::idle_time_limit = 600; // 10 minutes
+uint32_t IdleLEDs::idle_time_limit_ = 600000; // 10 minutes
 uint32_t IdleLEDs::start_time_ = 0;
 
+uint16_t IdleLEDs::idle_time_limit() {
+  return uint16_t(idle_time_limit_ / 1000);
+}
+
+void IdleLEDs::set_idle_time_limit(uint16_t new_limit) {
+  idle_time_limit_ = (uint32_t)new_limit * 1000;
```

Comment:
Might it be better to simply make `new_limit` parameter a `uint32_t`?

Reply:
I made it `uint16_t` to match the data type of the original, public property, but there's no reason it needs to stay that way. Thanks.